app/javascript/components/miq-data-table/miq-table-cell.jsx

```diff
@@ -100,15 +101,13 @@ const MiqTableCell = ({
 
   /** Function to execute a button click event
    * 'callbackAction' is added to row so that, the event can be executed within the parent component. eg: SettingsCompanyTags
-   * the onClick property executes a javascript function. Eg: "miqOrderService(#{record.id}); in catalogs_helper.rb"
-   * ToDo: 'onClick' can be removed and make use of the 'callbackAction', so that we can avoid the 'eval' feature.
   */
   const cellButtonEvent = (item, event) => {
     if (item.callback) {
       onCellClick({ ...row, callbackAction: item.callback }, CellAction.buttonCallback, event);
     } else {
       // eslint-disable-next-line no-unused-expressions
-      (item.onclick ? eval(item.onclick) : undefined);
+      (item.onclick ? customOnClickHandler(item.onclick) : undefined);
```

Comment:
Can this just more simply be `item.onclick.call()`, then just set the onclick in the various places to a function?

Reply:
yes, this should also work.